cookbook/serializer.rst

```diff
@@ -48,23 +46,48 @@ it in your configuration:
         $container->loadFromExtension('framework', array(
             // ...
             'serializer' => array(
-                'enabled' => true
+                'enabled' => true,
             ),
         ));
 
+Using the Serializer Service
+----------------------------
+
+Once enabled, the ``serializer`` service can be injected in any service where
+you need it or it can be used in a controller like the following::
+
+    // src/AppBundle/Controller/DefaultController.php
+    namespace AppBundle\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+    class DefaultController extends Controller
+    {
+        public function indexAction()
+        {
+            $serializer = $this->get('serializer');
+
+            // ...
+        }
+    }
+
 Adding Normalizers and Encoders
 -------------------------------
 
+.. versionadded:: 2.7
+    The :class:`Symfony\\Component\\Serializer\\Normalizer\\ObjectNormalizer`
+    is enabled by default in Symfony 2.7. In prior versions, you need to load
+    your own normalizer.
+
 Once enabled, the ``serializer`` service will be available in the container
 and will be loaded with two :ref:`encoders<component-serializer-encoders>`
 (:class:`Symfony\\Component\\Serializer\\Encoder\\JsonEncoder` and
-:class:`Symfony\\Component\\Serializer\\Encoder\\XmlEncoder`)
-but no :ref:`normalizers<component-serializer-normalizers>`, meaning you'll
-need to load your own.
+:class:`Symfony\\Component\\Serializer\\Encoder\\XmlEncoder`) and the
+:ref:`Symfony\\Component\\Serializer\\Normalizer\\ObjectNormalizer normalizer <component-serializer-normalizers>`.
```

Comment:
This will look a bit weird when being rendered. I would simply use "ObjectNormalizer" as the ref's label.